services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/DownloadDirectoryRequest.java

```diff
@@ -84,6 +86,14 @@ public String bucket() {
         return Optional.ofNullable(delimiter);
     }
 
+    /**
+     * @return the optional filter
+     * @see Builder#filter(DownloadFilter) 
+     */
+    public Optional<DownloadFilter> filter() {
+        return Optional.ofNullable(filter);
```

Comment:
Should we just return `(remote, local) -> true;` if it's null? https://github.com/aws/aws-sdk-java-v2/blob/master/services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/UploadDirectoryOverrideConfiguration.java#L82

Reply:
Good point, will do.